File: PartnerApi/C#/b2_authorize_account.cs
using System.Net.Http.Headers;
using System.Text;

class Program
{
    static readonly HttpClient Http = new();

    static async Task Main()
    {
        const string masterApplicationKeyId = ""; // Obtained from your B2 account page
        const string masterApplicationKey = ""; // Obtained from your B2 account page

        Http.DefaultRequestHeaders.Authorization =
            new AuthenticationHeaderValue("Basic",
                Convert.ToBase64String(Encoding.UTF8.GetBytes($"{masterApplicationKeyId}:{masterApplicationKey}")));

        var json = await Http.GetStringAsync("https://api.backblazeb2.com/b2api/v4/b2_authorize_account");

        Console.WriteLine(json);
    }
}

File: PartnerApi/Ruby/b2_authorize_account.rb
# frozen_string_literal: true

require 'net/http'
require 'uri'
require 'base64'

master_application_key_id = ''
master_application_key = ''

auth = Base64.strict_encode64("#{master_application_key_id}:#{master_application_key}")

request = Net::HTTP::Get.new(URI('https://api.backblazeb2.com/b2api/v4/b2_authorize_account'))
request['Authorization'] = "Basic #{auth}"

response = Net::HTTP.start(uri.host, uri.port, use_ssl: true) do |http|
  http.request(request)
end

puts response.body
